### PR TITLE
Add option to specify additional scope values

### PR DESCRIPTION
**Pull request by saruniitr**
## Change description
Default scope is 'read write', some apps need additional scope setup in order to access services, add a setting to specify additional scope values.

Our tool access Data Hub API which requires additional scope to be set. This PR allows to specify this using a settings variable STAFF_SSO_SCOPE.

### CODE REVIEW

**Comment by sdonk** [comment]
Can you please update the version number, the changelog and fix the tests?

File: authbroker_client/utils.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 TOKEN_URL = urljoin(settings.AUTHBROKER_URL, '/o/token/')
 AUTHORISATION_URL = urljoin(settings.AUTHBROKER_URL, '/o/authorize/')
 TOKEN_CHECK_PERIOD_SECONDS = 60
-SCOPE = 'read write'
+SCOPE = settings.get('STAFF_SSO_SCOPE', 'read write')
```
Comment (richtier) [inline authbroker_client/utils.py]: `getattr(settings, 'AUTHBROKER_STAFF_SSO_SCOPE', 'read write')`

Comment (saruniitr) [reply authbroker_client/utils.py]: ok

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,18 @@
 # Changelog
 
+## [2.0.1](https://pypi.org/project/django-staff-sso-client/2.0.1/) (2020-03-25)
```
Comment (sdonk) [inline CHANGELOG.md]: Can you please change it to `2.1.0`, the third number is a bugfix version of the major number

**Comment by sdonk** [comment]
Can you please update the version in here https://github.com/uktrade/django-staff-sso-client/blob/master/authbroker_client/version.py#L1 and add a test to check that if the scope is passed in the settings, then the client gets the right scope?

File: authbroker_client/version.py [hunk 3]
```diff
@@ -1 +1 @@
-__version__ = '2.0.1'
+__version__ = '2.0.2'
```
Comment (sdonk) [inline authbroker_client/version.py]: this needs to be 2.1.0

File: setup.py [hunk 4]
```diff
@@ -49,7 +49,7 @@ def get_version():
     ],
     extras_require={
         'test': [
-            'pytest',
+            'pytest==5.3.5',
```
Comment (sdonk) [inline setup.py]: Can you please add a comment of why we're pinning this

File: CHANGELOG.md (hunk 2 above)
Comment (saruniitr) [reply CHANGELOG.md]: done

File: authbroker_client/version.py (hunk 3 above)
Comment (saruniitr) [reply authbroker_client/version.py]: done

File: setup.py (hunk 4 above)
Comment (saruniitr) [reply setup.py]: added a comment explaining the reason

**Pull request merged**